x-pack/plugins/watcher/public/sections/watch_edit/components/json_watch_edit.tsx

```diff
@@ -138,9 +133,7 @@ const JsonWatchEditUi = ({
           watchActions={watchActions}
         />
       )}
-      {selectedTab === WATCH_EDIT_TAB && (
-        <JsonWatchEditForm urlService={urlService} licenseService={licenseService} />
-      )}
+      {selectedTab === WATCH_EDIT_TAB && <JsonWatchEditForm licenseService={licenseService} />}
```

Comment:
rather than passing this in as a prop, could you just import `LicenseServiceContext` directly from the `JsonWatchEditForm` component?

Reply:
good point, fixed.